contentcuration/contentcuration/migrations/0141_user_soft_delete.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.14 on 2022-10-19 09:41
+import django.db.models.deletion
+import django.utils.timezone
+from django.conf import settings
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contentcuration', '0140_delete_task'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='deleted_at',
```

Comment:
If we have `UserHistory`, maybe we don't need a timestamp here but what you originally had? Since we can determine when it was `performed_at`

Reply:
Discussion resolved on slack and pushed commits that addresses the concern